### PR TITLE
Globalize using strong-globalize

### PR DESCRIPTION
**Pull request by gunjpan**
Connect to strongloop/loopback/issues/2422

### CODE REVIEW

File: lib/rest-builder.js [hunk 1]
```diff
@@ -588,7 +589,7 @@ RequestBuilder.prototype.invoke = function(parameters, cb) {
  */
 RequestBuilder.prototype._request = function(uri, options, cb) {
   if (typeof uri === 'undefined') {
-    throw new Error('undefined is not a valid uri or options object.');
+    throw new Error(g.f('undefined is not a valid uri or options object.'));
```
Comment (superkhau) [inline lib/rest-builder.js]: {{undefined}} {{uri}} {{options}}

File: package.json [hunk 2]
```diff
@@ -11,10 +11,9 @@
   ],
   "main": "index.js",
   "scripts": {
-    "lint": "eslint .",   
+    "lint": "eslint .",
     "test": "mocha --timeout 30000 test/*test.js",
     "posttest": "npm run lint"
-
```
Comment (superkhau) [inline package.json]: unrelated changes above, but are nitpicks.

**Comment by superkhau** [comment]
One line needs resolution, other comment is a nitpick. Otherwise LGTM.

File: package.json (hunk 2 above)
Comment (gunjpan) [inline package.json]: That was update by `npm install strong-globalize --save`. And doesn't need to be reverted back.

**Comment by gunjpan** [comment]
@slnode test please

**Comment by gunjpan** [comment]
@slnode test please

File: lib/rest-connector.js [hunk 3]
```diff
@@ -315,7 +317,8 @@ RestConnector.prototype.responseHandler = function(model, callback, many) {
         callback(null, result);
       }
     } else {
-      callback && callback('Error response: ' + response.statusCode, body);
+      var err = g.f('Error response: %s %j', response.statusCode, body);
```
Comment (superkhau) [inline lib/rest-connector.js]: %d for statusCode which is a number I assume?

File: lib/rest-connector.js [hunk 4]
```diff
@@ -271,7 +272,8 @@ RestConnector.prototype.create = function create(model, data, callback) {
     if (response.statusCode === 200 || response.statusCode === 201) {
       callback && callback(null, body.id);
     } else {
-      callback && callback('Error response: ' + response.statusCode, body);
+      var err = g.f('Error response: %s %j', response.statusCode, body);
```
Comment (superkhau) [inline lib/rest-connector.js]: ditto

**Comment by superkhau** [comment]
Two minor comments, otherwise LGTM.

**Pull request merged**